.azure-pipelines/LinuxAnalyzerTest.yml

```diff
@@ -43,6 +43,13 @@ jobs:
       mkdir -p $HOME/go/bin
       curl https://raw.githubusercontent.com/golang/dep/v$GO_DEP_VERSION/install.sh | sh
 
+      # Install 'CocoaPods'. As https://github.com/CocoaPods/CocoaPods/pull/10609 is needed but not yet released, make a
+      # build from the latest revision of the master branch.
+      git clone https://github.com/CocoaPods/CocoaPods.git
```

Comment:
How about downloading exactly the revision we need (so the download does not change all the time), and without the Git history: https://github.com/CocoaPods/CocoaPods/archive/9461b346aeb8cba6df71fd4e71661688138ec21b.zip (This is the merge commit for the wanted PR.)

Reply:
I'd like to leave this improvement up to a follow-up PR as this does cause me quite a bit: context switch and testing time.

Reply:
Ok.

Reply:
See #4204.